.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+git:
+  depth: false
+
+dist: trusty
+sudo: required
+
+matrix:
+  include:
```

Comment:
Add another job for just linting, it can run `npm run lint`

Reply:
👍